### PR TITLE
Add search on Event

### PR DESCRIPTION
**Pull request by qifeng-bai**
1, Add a Event tab search form
2, Create a Event section on an occurrence page

### CODE REVIEW

File: grails-app/i18n/messages_en.properties [hunk 1]
```diff
@@ -546,7 +561,7 @@ facet.country_conservation=Country conservation
 facet.data_provider_uid=Data provider
 facet.data_resource=Data resource
 facet.data_resource_uid=Data resource
-facet.dataset_name=Dataset name
+facet.dataset_name=Dataset / Survery name
```
Comment (djtfmartin) [inline grails-app/i18n/messages_en.properties:564]: Survery ?

File: grails-app/i18n/messages_en.properties [hunk 2]
```diff
@@ -297,7 +312,7 @@ list.sortwidgets.span02 = page
 list.sortwidgets.sort.label = sort
 list.sortwidgets.sort.option01 = Best match
 list.sortwidgets.sort.option02 = Taxon name
-list.sortwidgets.sort.option03 = Common name
+list.sortwidgets.sort.option03 = Common namegit
```
Comment (djtfmartin) [inline grails-app/i18n/messages_en.properties:315]: typo

File: grails-app/conf/plugin.groovy [hunk 3]
```diff
@@ -95,7 +95,7 @@ clubRoleForHub = "ROLE_ADMIN"
 // whether map or list is the default tab to show - empty for list and "mapView" for map
 defaultListView = "" //  'mapView' or 'listView'
 dataQualityChecksUrl = "https://docs.google.com/spreadsheet/pub?key=0AjNtzhUIIHeNdHJOYk1SYWE4dU1BMWZmb2hiTjlYQlE&single=true&gid=0&output=csv"
-dwc.exclude = "dataHubUid,dataProviderUid,institutionUid,year,month,day,modified,left,right,provenance,taxonID,preferredFlag,outlierForLayers,speciesGroups,associatedMedia,images,userQualityAssertion,speciesHabitats,duplicationType,taxonomicIssues,subspeciesID,nameMatchMetric,sounds"
+dwc.exclude = "dataHubUid,dataProviderUid,institutionUid,year,month,day,modified,left,right,provenance,taxonID,preferredFlag,outlierForLayers,speciesGroups,associatedMedia,images,userQualityAssertion,speciesHabitats,duplicationType,taxonomicIssues,subspeciesID,nameMatchMetric,sounds,datasetName, fieldNumber"
```
Comment (djtfmartin) [inline grails-app/conf/plugin.groovy:98]: space before `fieldNumber`

File: grails-app/i18n/messages_en.properties (hunk 2 above)
Comment (qifeng-bai) [reply grails-app/i18n/messages_en.properties:315]: Done

File: grails-app/i18n/messages_en.properties (hunk 1 above)
Comment (qifeng-bai) [reply grails-app/i18n/messages_en.properties:564]: done

File: grails-app/conf/plugin.groovy (hunk 3 above)
Comment (qifeng-bai) [reply grails-app/conf/plugin.groovy:98]: done

**Pull request merged**